MapTopic:              '/voxel_map'

InitMapFirst:            true

TargetTopic:            '/move_base_simple/goal'

DilateRadius:           0.5

VoxelWidth:              0.5

MapBound:                [-25.0, 25.0, -25.0, 25.0, 0.0, 5.0]

